spark.yaml

```diff
@@ -0,0 +1,157 @@
+    aliases:
+      - GHSA-g2fg-mr77-6vrm
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: fix-not-planned
+        data:
+          note: Spark v3.5.0 is incompatible with higher versions of libthrift. https://github.com/apache/spark/pull/34878
+
+  - id: CVE-2020-8908
+    aliases:
+      - GHSA-5mg8-w23w-74h3
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: fix-not-planned
+        data:
+          note: This relates to guava v14.0.1, which the upstream project has a hard dependency on. Uprading causes complication issues, and there are no plans for upstream to upgrade. https://issues.apache.org/jira/browse/SPARK-23897
+
+  - id: CVE-2021-22569
+    aliases:
+      - GHSA-wrvw-hg22-4m67
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
+
+  - id: CVE-2021-22570
+    aliases:
+      - GHSA-77rm-9x9h-xj3g
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
+
+  - id: CVE-2021-31684
+    aliases:
+      - GHSA-fg2v-w576-w4v3
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: json-smart classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
+
+  - id: CVE-2022-3171
+    aliases:
+      - GHSA-h4h5-3hr4-j3g2
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
+
+  - id: CVE-2022-3509
+    aliases:
+      - GHSA-g5ww-5jh7-63cx
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
+
+  - id: CVE-2022-3510
+    aliases:
+      - GHSA-4gg5-vx3j-xwc7
+    events:
+      - timestamp: 2024-02-02T22:15:34Z
+        type: false-positive-determination
+        data:
+          type: vulnerable-code-not-included-in-package
+          note: protobuf-java classes are not included in runtime jars, but its metadata is still listed in META-INF pom.xml and properties files
```

Comment:
I'm not sure I agree with this - I have found the protobuf classes in the shaded jar

Reply:
Resolving as we paired up earlier and walked through some of the items, including this one :)